### PR TITLE
RFD 01111 - Support connection testers when per-session MFA is enabled

### PR DESCRIPTION
**Pull request by kimlisa**
rendered: https://github.com/gravitational/teleport/blob/5af70fdb6d9a81d2da6ba0676e4ce280a0bb6b18/rfd/0111-support-connection-testers-with-per-session-mfa.md

### CODE REVIEW

File: rfd/0111-user-cert-accepts-mfa-authn-response.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+authors: Lisa Kim (lisa@goteleport.com)
+state: draft
+---
+
+# RFD 0111 - GenerateUserCerts accepts and validates a MFA authentication response
```
Comment (zmb3) [inline rfd/0111-user-cert-accepts-mfa-authn-response.md:6]: This title seems more like an implementation detail than a description of the proposed change.

How about something like:

```suggestion
# RFD 0111 - Support connection testers when per-session MFA is enabled
```

Comment (zmb3) [reply rfd/0111-user-cert-accepts-mfa-authn-response.md]: Same goes for the PR title :-)

Comment (kimlisa) [reply rfd/0111-user-cert-accepts-mfa-authn-response.md]: ah right thanks!

**Review by codingllama — APPROVED** [review]
Thanks for the RFD, Lisa. Short and sweet.

**Review by marcoandredinis — APPROVED** [review]

**Pull request merged**